### ISSUE TITLE
duplication of effort with Pix

### ISSUE THREAD

**Issue by gordon-morehouse** [linuxmint/xviewer#33]
Both xviewer and Pix are part of the Linux Mint project. It would be good to officially define the scope of xviewer vs Pix to avoid duplication of effort.

For example, there is already a slideshow feature in both. Why?

I find myself tempted to submit a pet feature request regarding the slideshow to both projects in case it gets done in one and not in the other. Even without having such an intent, over time users are going to submit duplicate feature requests, and developers may spend time writing duplicate features.

I suggest the following:

  * xviewer should be kept as simple as possible
  * extended features (slideshows, tagging, whatever) be kept in Pix
  * basic, commonly used features (crop, rotate, export as) be common to both for convenience, *after* it has been carefully thought through

**Comment by JosephMcc** [comment]
I agree with you to an extent. However these came from existing applications that already had those features built in. I don't really see the point in removing until/if we have a good reason. For new features it would be a bit different.

**Comment by jversusk** [comment]
I can think of one way Pix and Xviewer can and need to be different. Currently there is no image viewer that automatically uses the file manager's sort order. Pix being both viewer and browser needs to have it's own sort options, independent of the file manager. Xviewer on the other hand is only a viewer, so it should ideally use whatever sort order the file manager has for the source folder.
